net/yt-dlp-ejs/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                yt-dlp-ejs
+github.setup        yt-dlp ejs 0.3.1
+github.tarball_from releases
+distname            yt_dlp_ejs-${github.version}
+revision            0
+categories          net
+
+homepage            https://github.com/yt-dlp/ejs
+
+maintainers         {@akierig fastmail.de:akierig} openmaintainer
+
+checksums           rmd160  a4c1381407cfb64df57bf76a508bf2a2d39ed36b \
+                    sha256  7f2119eb02864800f651fa33825ddfe13d152a1f730fa103d9864f091df24227 \
+                    size    33805
+
+description         ${name} is an external JavaScript utility for \
+                    yt-dlp that supports multiple JS runtimes.
+long_description    {*}${description}. It enables yt-dlp to solve \
+                    JavaScript challenges for YouTube and other sites.
+
+supported_archs     noarch
+platforms           {darwin any}
+license             public-domain
+
+variant python310 conflicts python311 python312 python313 description {Use Python 3.10} {}
+variant python311 conflicts python310 python312 python313 description {Use Python 3.11} {}
+variant python312 conflicts python310 python311 python313 description {Use Python 3.12} {}
+variant python313 conflicts python310 python311 python312 description {Use Python 3.13} {}
+
+if {![variant_isset python310] && ![variant_isset python311] && ![variant_isset python312] && ![variant_isset python313]} {
+    default_variants +python313
+}
+
+if {[variant_isset python310]} {
+    python.default_version  310
+} elseif {[variant_isset python311]} {
+    python.default_version  311
+} elseif {[variant_isset python312]} {
+    python.default_version  312
+} elseif {[variant_isset python313]} {
+    python.default_version  313
+}
+
+python.pep517_backend   hatch
+
+depends_build-append    port:py-hatch-vcs
```

Comment:
py-hatch-vcs, like almost all other py-* ports, doesn't install anything; it just depends on another port that installs something. You should depend on *that* port (e.g. py313-hatch-vcs) and ensure that that python (e.g. python3.13) is used.